app/debrief_gui.py

```diff
@@ -38,6 +38,8 @@ def __init__(self, services):
         self.report_section_names = dict()
         self.loaded_report_sections = False
 
+        rl_settings.trustedHosts = [BaseWorld.get_config('host')]
```

Comment:
Does this work if the `host` config is set to `0.0.0.0`? I glanced at the reportlab source code and it seems like it might throw errors: https://hg.reportlab.com/hg-public/reportlab/file/ea083dcb248b/src/reportlab/lib/utils.py#l706

Reply:
Debrief definitely still works. I would imagine that if the plugin is making a request to something like `192.168.0.30` and the host was set to `0.0.0.0`, that the match would fail.

We have access to `request.host` in `download_pdf()`, though. Should we add that host to `trustedHosts` as well?

Reply:
Unless we're doing some other higher-level host header verification, I think injecting the `request.host` into the trusted/allowed hosts configuration will counteract this protection.  Hmm. What if we exposed a new config option for this?